lib/logflare/backends/adaptor/clickhouse_adaptor/connection_manager.ex

```diff
@@ -148,27 +90,13 @@ defmodule Logflare.Backends.Adaptor.ClickhouseAdaptor.ConnectionManager do
   end
 
   @impl true
-  def init(args) do
+  def init(%Backend{id: backend_id}) do
```

Comment:
if we could just set this to backend_id if thats the case.
less terms stored in the process dict.

Reply:
Done